src/builtin/tables.scrbl

```diff
@@ -29,12 +29,32 @@
 
 @docmodule["tables" #:noimport #t #:friendly-title "Tables"]{
 
-  Tables consist of a sequence of zero or more @italic{rows}, which each
-  contain an equal amount of entries in named @italic{columns}.
-
+There are many examples of tables in computing, with
+spreadsheets being the most obvious.
+                                                             
+A @pyret{Table} is made up of @bold{rows} and @bold{columns}. Each
+column has a name.  Each row has an equal number of columns,
+in the same order.  Each column may be assigned a type via an
+annotation, and each entry in that column will be checked against
+the annotation.
+
+@margin-note{Note that as of June 2017, the @pyret{tables} implementation in
+is rather minimal.  It meets the basic requirements of the curricula that
+use it, but lacks many features that one might reasonably expect from a
+complete table library.
+
+Additionally, after the first year of using @pyret{tables}, the Pyret team
+is considering a new table design and interface that will make it easier
+to add features in the future, but in the meantime puts small changes
+to the current design on hold.
+
+Tabular data is here to stay in Pyret and Pyret-related curricula, so
+count on further development of @pyret{tables}.}
+  
   @section[#:tag "s:tables"]{Creating Tables}
 
-Tables are created with a @pyret{table:} expression, which lists any number of
+A simple @pyret{Table} can be "manually" created with a @pyret{table:}
```

Comment:
In fact, being able to create literal tables is _essential_ for a curriculum (like ours) in which students are expected to write examples and tests. You don't want to have to create a dozen files on GDrive, all fragmented: you won't even be able to see the input and output in one place, depending on how you set it up, which offers the program's reader no insight. Whereas pithy literal tables are vital for explaining what a program or function is trying to do. So we should be _encouraging_ this.